extensions/guacamole-auth-sso/modules/guacamole-auth-sso-saml/pom.xml

```diff
@@ -122,20 +122,20 @@
         <dependency>
             <groupId>com.fasterxml.woodstox</groupId>
             <artifactId>woodstox-core</artifactId>
-            <version>5.4.0</version>
+            <version>6.6.0</version>
         </dependency>
 
         <!-- Apache XML Security for Java (see exclusions for java-saml) -->
         <dependency>
             <groupId>org.apache.santuario</groupId>
             <artifactId>xmlsec</artifactId>
-            <version>2.2.6</version>
+            <version>4.0.2</version>
```

Comment:
Are these newer versions known to be binary-compatible with the versions being overridden?

Reply:
woodstox-core 6.6.0 has full compatibility with 5 https://github.com/FasterXML/woodstox/wiki/Woodstox-Release-6.6

xmlsec I couldn't find any information on compatibility between 4 and 2, but I was able to successfully sign in with saml using these updated versions